sci-libs/calculix-cgx/calculix-cgx-2.10.ebuild

```diff
@@ -20,28 +20,38 @@ KEYWORDS="~amd64 ~x86"
 IUSE="doc examples"
 
 RDEPEND="media-libs/mesa[nptl]
-	>=media-libs/freeglut-1.0"
+	>=media-libs/freeglut-1.0
+	virtual/opengl
+	x11-libs/libX11
+	x11-libs/libXmu
+	x11-libs/libXi
+	x11-libs/libXext
+	x11-libs/libXt
+	x11-libs/libSM
+	x11-libs/libICE"
 DEPEND="${RDEPEND}
 	doc? ( app-text/ghostscript-gpl )"
 
 S=${WORKDIR}/CalculiX/${MY_P}/src/
 
+PATCHES() {
+	"${FILESDIR}"/01_${MY_P}_Makefile_custom_cxx_flags.patch
+	"${FILESDIR}"/02_${MY_P}_menu_fix-freeglut_2.8.1.patch
+}
+
 src_unpack() {
 	default
 	cp "${DISTDIR}/${MY_P}.pdf" "${S}"
```

Comment:
Also you may drop this and dodoc "${DISTDIR}/${MY_P}.pdf" right from src_install
Otherwise you should check the "doc" USE here.